src/main/java/io/vlingo/schemata/Bootstrap.java

```diff
@@ -26,14 +25,15 @@
   private final Server server;
   private final World world;
 
-  public Bootstrap() {
+  public Bootstrap() throws Exception {
     world = World.startWithDefaults("vlingo-schemata");
 
     NoopDispatcher dispatcher = new NoopDispatcher();
-    Journal<String> journal = Journal.using(world.stage(), InMemoryJournalActor.class, dispatcher);
 
-    SourcedTypeRegistry registry = new SourcedTypeRegistry(world);
-    EntryAdapters.register(registry, journal);
+    SchemataObjectStore schemataObjectStore = new SchemataObjectStore("SA", "");
```

Comment:
In the spirit of immutability where possible, we have standardized on using `final`. It's noisy but we will never have `val` in Java.

Reply:
you meant I have to add to class definition and method parameters ? I tried to do it but forgot in some places as for now I have to think explicitly about it :)

Reply:
@tjaskula In these specific places I mean this:

```
final SourcedTypeRegistry registry = new SourcedTypeRegistry(world);
...
final SchemataObjectStore schemataObjectStore = new SchemataObjectStore("SA", "");
```

But, yes, it is best to use final everywhere possible. You know how `val` works in Scala. This is to achieve the same constraint as `val` but it's way more verbose here.

Reply:
Ok, got it. I'm not yet familiar with the syntax but will apply it forward on